shell/platform/linux/fl_view.cc

```diff
@@ -584,6 +586,7 @@ static void fl_view_dispose(GObject* object) {
   g_clear_object(&self->engine);
   g_clear_object(&self->accessibility_plugin);
   g_clear_object(&self->keyboard_manager);
+  g_signal_handler_disconnect(self->keymap, self->keymap_keys_changed_cb_id);
```

Comment:
Hey, I've started seeing warnings like
```
(bug:74019): GLib-GObject-WARNING **: 11:15:08.697: ../../../gobject/gsignal.c:2731: instance '0x55e1c3ea0200' has no handler with id '255'
```
when closing Flutter apps because GObject's dispose() method may be called multiple times. Here's a fix: https://github.com/flutter/engine/pull/35490